doc/manpages/man8/netatalk.8.md

```diff
@@ -32,11 +41,11 @@ normally started at boot time by an init system.
 
 SIGTERM
 
-> Stop Netatalk service, AFP and CNID daemons
+> Stop the Netatalk AFP and CNID daemons.
 
 SIGHUP
 
-> Sending a *SIGHUP* will cause the AFP daemon reload its configuration
+> Sending a *SIGHUP* will cause the AFP daemon to reload its configuration
```

Comment:
Given the multiple daemos in play here, this phrasing is still a bit confusing, I think, and it might be better to specify the daemon(s) by name - is it only afpd that gets its configuration reloaded, or is it all child daemons of netatalk?